Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+    ssl_certificates = domain_result['ssl_info']
+    redirects_to = domain_result['redirect']
+
+    domain_tools_context = {
+        "Name": domain,
+        "LastEnriched": datetime.now().strftime('%Y-%m-%d'),
+        "Analytics": {
+            "OverallRiskScore": overall_risk_score,
+            "ProximityRiskScore": proximity_risk_score,
+            "ThreatProfileRiskScore": {"RiskScore": threat_profile_risk_score,
+                                       "Threats": threat_profile_threats,
+                                       "Evidence": threat_profile_evidence},
+            "WebsiteResponseCode": website_response,
+            "Alexa Rank": alexa_rank,
+            "Tags": tags
+        },
+        "Identity": {
+            "RegistrantName": registrant_name,
+            "RegistrantOrg": registrant_org,
+            "RegistrantContact": registrant_contact,
+            "SOAEmail": soa_email,
+            "SSLCertificateEmail": ssl_email,
+            "AdminContact": admin_contact,
+            "TechnicalContact": technical_contact,
+            "BillingContact": billing_contact,
+            "EmailDomains": email_domains,
+            "AdditionalWhoisEmails": additional_whois_emails
+        },
+        "Registration": {
+            "DomainRegistrant": domain_registrant,
+            "RegistrarStatus": registrar_status,
+            "DomainStatus": domain_status,
+            "CreateDate": create_date,
+            "ExpirationDate": expiration_date
+        },
+        "Hosting": {
+            "IPAddresses": ip_data,
+            "IPCountryCode": ip_country_code,
+            "MailServers": mx_servers,
+            "SPFRecord": spf_info,
+            "NameServers": name_servers,
+            "SSLCertificate": ssl_certificates,
+            "RedirectsTo": redirects_to,
+            "GoogleAdsenseTrackingCode": google_adsense,
+            "GoogleAnalyticTrackingCode": google_analytics
+        }
+    }
+
+    domain_context = {
+        "Name": domain,
+        "DNS": ip_addresses,
+        "Vendor": "DomainTools",
+        "CreationDate": create_date,
+        "RiskScore": overall_risk_score,
+        "DomainStatus": domain_status,
+        "ExpirationDate": expiration_date,
+        "NameServers": domain_name_servers,
+        "Registrant": domain_registrant_contact,
+        "DomainTools": domain_tools_context
+    }
+    if overall_risk_score and overall_risk_score >= RISK_THRESHOLD:
+        domain_context["Malicious"] = {
+            "Vendor": 'DomainTools',
+            "Description": threat_profile_evidence
+        }
+
+    return domain_context
+
+
+def format_domain_profile(response):
+    domain_result = response['results'][0]
+    domain_context = create_context_output(domain_result)
+    outputs = {"Domain(val.Name == obj.Name)": domain_context}
+    convert_empty_to_null(outputs)
+
+    human_readable = '### DomainTools Profiled {}. See context data for more info.'.format(domain_result['domain'])
```

Comment:
Users should not look at the context data for outputs, let's return a table here.